scout/load/panel.py

```diff
@@ -77,13 +80,26 @@ def load_panel(panel_path, adapter, **kwargs):
         display_name = display_name or existing_panel["display_name"]
         institute = institute or existing_panel["institute"]
 
+    # Check if maintainers exist in the user database
+    maintainer = kwargs.get("maintainer")
+    if maintainer is not None:
+        scout_users = adapter.users()
```

Comment:
Will not this return a iterable och user objects? If that's the case the check will fail since we ask if a string is in a list of dicts

Reply:
Yeah, pretty much! That was the reason for the list comprehension in the other function.. Fixed.

Reply:
..and added a test for it as well.